src/components/NewTransaction.js

```diff
@@ -3,6 +3,14 @@ import React from "react";
 const NewTransaction = () => {
   return <div>
     <h2>New transaction</h2>
+    <label for="title">Title</label>
+    <input id="title" type="text" />
+    <label for="amount">Amount</label>
```

Comment:
setting the 'for' attribute is the standard way to bind the field to its label, but two points there:
1- I believe in React, you should use HTMLFor, not `for` .. double check that
2- there is an asier trick to get this done without the need for id and for attributes, by wrapping the input element with the label, i.e:
`<label> label text <input type="text"/> </label>`

Reply:
You are right, I should use `htmlFor` instead of `for`.
However, I decided to change it the second way you proposed.